neopixel.py

```diff
@@ -199,6 +199,30 @@ def set_pixel(self, pixel_num, rgb_w, how_bright=None):
         else:
             self.pixels[pixel_num] = pix_value
 
+    def get_pixelRGB(self, pixel_num):
+        """
+        Get red, green and blue value of pixel on position <pixel_num>
+
+        :param pixel_num: Index of pixel to be set or slice object representing multiple leds
+        :return rgb_w: Tuple of form (r, g, b) or (r, g, b, w) representing color to be used
+        """
+        balance = self.pixels[pixel_num]
+        if self.W_in_mode:
+            w = balance & 255
+            balance = (balance-w) >> 8
+        b = balance & 255
+        balance = (balance-b) >> 8
+        r = balance & 255
+        g = ((balance-r) >> 8) & 255
+        red = int(r * 255 / self.brightness() )
+        green = int(g * 255 / self.brightness() )
+        blue = int(b * 255 / self.brightness() )
+        if self.W_in_mode:
+            white = int(w * 255 / self.brightness() )
+            return (red,green,blue,white)
+        else:
+            return (red,green,blue)
```

Comment:
This is OK. It's returning the value that was passed, with brightness reversed, this "reversal" of brightness should also be specified in doscstring.